homeassistant/components/mqtt/climate.py

```diff
@@ -858,7 +824,7 @@ async def async_set_preset_mode(self, preset_mode: str) -> None:
             )
 
             if self._optimistic_preset_mode:
-                self._preset_mode = preset_mode if preset_mode != PRESET_NONE else None
+                self._attr_preset_mode = preset_mode
```

Comment:
This is a code change. Was it planned?

Reply:
The logic was in the property code to return the preset_mode or PRESET_NONE. With removing the property the code here is less complex now.